index.html

```diff
@@ -6,15 +6,43 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio Microverse</title>
     <link rel="stylesheet" href="styles/styles.css" />
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <nav>
-        <ul>
-          <li class="navbar-link"><a href="#">Portfolio</a></li>
-          <li class="navbar-link"><a href="#">About</a></li>
-          <li class="navbar-link"><a href="#">Contact</a></li>
-        </ul>
-    </nav>
+
+    <header>
+      <!-- NAVBAR -->
+  
+      <nav>
+          <ul>
+            <li class="navbar-link"><a href="#">Welcome</a></li>
+            <li class="navbar-link"><a href="#"><img src="icons/menu-icon.png" alt="Icon"></a></li>
+          </ul>
+      </nav>
+  
+      <!-- HEADLINE -->
+  
+      <div class="headline">
+        <h1>
+          <span class="dark-blue">Hey there. I'm Javier</span>
+          <span class="green">I'm a software developer</span>
+        </h1>
+        <p>
+          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+        </p>
+        <div class="social-media-headline">
+          <a href="#"><img src="icons/Vector.png" alt="Github"></a>
+          <a href="#"><img src="icons/Frame.png" alt="Linkedin"></a>
+          <a href="#"><img src="icons/Vector-hand.png" alt="Icon"></a>
+          <a href="#"><img src="icons/twitter.png" alt="Twitter"></a>
+          <a href="#"><img src="icons/m-icon.png" alt="Icon"></a>
+        </div>
```

Comment:
- [ ] Great job using `a` links for your social icons! However, it would be better to place them inside `ul` and `li` as this is a better practice in HTML.

Reply:
Great, I would like to ask why is it better to put them inside a ul and a li item.

Reply:
@Javivy sure! It's all about semantics:

Navigation is a very important aspect of a website design and the `<ul>` element combined with CSS makes your navigation menus semantic (because, after all, a navigation menu is a list of links) as well as beautiful. Also, by convention, an unordered list for your navigation menu has been the accepted markup.

[Source](https://www.webfx.com/blog/web-design/20-html-best-practices-you-should-follow/)